### PR TITLE
added solution for homework

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

**Comment by AnyaRatusnyak** [comment]
Hello ) Have a nice day !
Screenshot_15

File: src/main/java/ticketsproject/config/MyCustomConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MyCustomConfiguration {
+    private static final Logger logger = LoggerFactory.getLogger(MyCustomConfiguration.class);
+
+    @Bean
+    @ConditionalOnProperty(name = "creationMyFirstConditionalBean", havingValue = "true")
+    public String ThisIsMyFirstConditionalBean() {
```
Comment (Alexmansar) [inline src/main/java/ticketsproject/config/MyCustomConfiguration.java]: fix name thisIs...

File: src/main/java/ticketsproject/dao/TicketDataBaseDao.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package ticketsproject.dao;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ticketsproject.model.BusTicket;
+import ticketsproject.model.TicketDataBase;
+
+import java.util.List;
+
+public interface TicketDataBaseDao extends JpaRepository<TicketDataBase, Long> {
```
Comment (Alexmansar) [inline src/main/java/ticketsproject/dao/TicketDataBaseDao.java]: TicketDataBaseRepository

File: src/main/java/ticketsproject/dao/UserDataBaseDao.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package ticketsproject.dao;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ticketsproject.model.TicketDataBase;
+import ticketsproject.model.UserDataBase;
+
+import java.util.List;
+
+public interface UserDataBaseDao extends JpaRepository<UserDataBase, Long> {
```
Comment (Alexmansar) [inline src/main/java/ticketsproject/dao/UserDataBaseDao.java]: same

File: src/main/java/ticketsproject/service/TicketDataBaseService.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package ticketsproject.service;
+
+import ticketsproject.model.TicketDataBase;
+
+public interface TicketDataBaseService {
+    TicketDataBase findById(Long id);
```
Comment (Alexmansar) [inline src/main/java/ticketsproject/service/TicketDataBaseService.java]: DataBase is that necessary?
in general in entity name))

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak Hi)) good job in general)) few comments and 1 general about class naming.

File: src/main/java/ticketsproject/config/MyCustomConfiguration.java (hunk 1 above)
Comment (AnyaRatusnyak) [reply src/main/java/ticketsproject/config/MyCustomConfiguration.java]: Ok)

File: src/main/java/ticketsproject/dao/TicketDataBaseDao.java (hunk 2 above)
Comment (AnyaRatusnyak) [reply src/main/java/ticketsproject/dao/TicketDataBaseDao.java]: Ok)

File: src/main/java/ticketsproject/service/TicketDataBaseService.java (hunk 4 above)
Comment (AnyaRatusnyak) [reply src/main/java/ticketsproject/service/TicketDataBaseService.java]: Do you mean to change name of entity ? Can I change it to TicketInfo ? I have already had Ticket and BusTicket entities in my project

File: src/main/java/ticketsproject/dao/UserDataBaseDao.java (hunk 3 above)
Comment (AnyaRatusnyak) [reply src/main/java/ticketsproject/dao/UserDataBaseDao.java]: Ok)

File: src/main/java/ticketsproject/service/TicketDataBaseService.java (hunk 4 above)
Comment (Alexmansar) [reply src/main/java/ticketsproject/service/TicketDataBaseService.java]: Yes) and the same for the User.. without this DataBase

Comment (AnyaRatusnyak) [reply src/main/java/ticketsproject/service/TicketDataBaseService.java]: Ok , thank you very much )

File: pom.xml [hunk 5]
```diff
@@ -25,7 +30,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>${postgresql.version}</version>
+            <scope>runtime</scope>
```
Comment (Alexmansar) [inline pom.xml]: why we have here changes?

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak 1 small comment

File: pom.xml (hunk 5 above)
Comment (AnyaRatusnyak) [reply pom.xml]: I thought It was optimization for the database. But I probably didn't understand it in the correct way. I'll fix it

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Good job) could be merged)

**Pull request merged**